app/src/main/java/ch/epfl/skysync/screens/admin/AdminFlightDetailsScreen.kt

```diff
@@ -23,38 +50,123 @@ fun AdminFlightDetailScreen(
 ) {
 
   val flight by viewModel.getFlight(flightId).collectAsStateWithLifecycle()
-  Scaffold(modifier = Modifier.fillMaxSize(), bottomBar = { AdminBottomBar(navController) }) {
-      padding ->
-    if (flight is ConfirmedFlight) {
-      ConfirmFlightDetailUi(
-          confirmedFlight = flight as ConfirmedFlight,
-          backClick = { navController.popBackStack() },
-          paddingValues = padding,
-          okClick = { navController.navigate(Route.ADMIN_HOME) },
-      )
-    } else {
-      FlightDetailUi(
-          backClick = { navController.popBackStack() },
-          deleteClick = {
-            viewModel.deleteFlight(flightId)
-            navController.navigate(Route.ADMIN_HOME)
-          },
-          editClick = { navController.navigate(Route.MODIFY_FLIGHT + "/${flightId}") },
-          confirmClick = { navController.navigate(Route.CONFIRM_FLIGHT + "/${flightId}") },
-          padding = padding,
-          flight = flight,
-          bottom = { editClick, confirmClick, deleteClick,
-            ->
-            FlightDetailBottom(editClick, confirmClick, deleteClick)
-          })
-    }
+  val showConfirmDialog = remember { mutableStateOf(false) }
+  if (showConfirmDialog.value) {
```

Comment:
Nitpick : you could use `val showConfirmDialog by remember { mutableStateOf(false) }` to avoid all the .value later in the code

Reply:
I will do that